Application/src/visuals/Draw.java

```diff
@@ -106,18 +133,10 @@ void init() {
     void renderContainer() {
         if (!DiagramView.inView(this)) return;
         GraphicsContext gc = canvas.getGraphicsContext2D();
```

Comment:
variable name. Suggestions: graphicsContext

Reply:
That wasn't changed in this pull request? So I don't think I'll change it.

Reply:
ops ye sorry